test/test_transforms_tensor.py

```diff
@@ -600,14 +492,112 @@ def test_random_perspective_save():
     (T.RandomGrayscale, {})
 ])
 def test_to_grayscale(device, Klass, meth_kwargs):
-
     tol = 1.0 + 1e-10
     _test_class_op(
         Klass, meth_kwargs=meth_kwargs, test_exact_match=False, device=device,
         tol=tol, agg_method="max"
     )
 
 
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('in_dtype', int_dtypes() + float_dtypes())
+@pytest.mark.parametrize('out_dtype', int_dtypes() + float_dtypes())
+def test_convert_image_dtype(device, in_dtype, out_dtype):
+    tensor, _ = _create_data(26, 34, device=device)
+    batch_tensors = torch.rand(4, 3, 44, 56, device=device)
+
+    in_tensor = tensor.to(in_dtype)
+    in_batch_tensors = batch_tensors.to(in_dtype)
+
+    fn = T.ConvertImageDtype(dtype=out_dtype)
+    scripted_fn = torch.jit.script(fn)
+
+    if (in_dtype == torch.float32 and out_dtype in (torch.int32, torch.int64)) or \
+            (in_dtype == torch.float64 and out_dtype == torch.int64):
+        with pytest.raises(RuntimeError, match=r"cannot be performed safely"):
+            _test_transform_vs_scripted(fn, scripted_fn, in_tensor)
+        with pytest.raises(RuntimeError, match=r"cannot be performed safely"):
+            _test_transform_vs_scripted_on_batch(fn, scripted_fn, in_batch_tensors)
```

Comment:
There used to be a `continue` here. It should now be converted into a `return` and this should fix the current failures :)

Reply:
@NicolasHug Sure,but I had added the pytest.xfail because you had mentioned this in the original issue to  convert the continue and the assertRaises into a pytest.xfail

Reply:
No worries, thanks for explaining this to me. Appreciate it :)